z3/snap.py

```diff
@@ -77,7 +77,7 @@ def __repr__(self):
 
     @property
     def is_full(self):
-        return self._metadata.get('is_full') == 'true'
```

Comment:
I suggest keeping maintaining the old format since there are snapshots in S3 with `is_full` metadata.
```python
# keep backwords compatibility for underscore metadata.
return 'true' in [self._metadata.get('is_full'), self._metadata.get('isfull')]
```

Reply:
Makes sense, amended that.